### PR TITLE
WEBAPP-410: Remove Question "Where are you" on city selection page

### PR DESCRIPTION
**Pull request by a-kud**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

image

### CODE REVIEW

File: src/locales.json [hunk 1]
```diff
@@ -209,8 +209,9 @@
       "searchCity": "البحث عن مكانك"
     },
     "de": {
-      "where": "Wo bist du?",
-      "searchCity": "Suche nach einem Ort"
+      "where": "",
```
Comment (maxammann) [inline src/locales.json]: You can remove this in the code and remove this key completely.

File: src/locales.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
         "search": "Suche",
         "events": "News",
         "extras": "Extras",
-        "landing": "Wo bist du?",
+        "landing": "Ortsauswahl",
```
Comment (maxammann) [inline src/locales.json]: The issue descrption was a bit unclear. Please also change this in the other languages. For languages like farsi or arabic you can omit the key currently.

Comment (maxammann) [reply src/locales.json]: If you omit a key it will fallback automatically to english

File: src/routes/landing/components/Heading.js [hunk 3]
```diff
@@ -3,9 +3,7 @@
 import React from 'react'
 
 import LocationBig from '../assets/LocationBig.svg'
-import type { TFunction } from 'react-i18next'
 import { withNamespaces } from 'react-i18next'
```
Comment (a-kud) [inline src/routes/landing/components/Heading.js]: Component should stay connected and this should remain here, right?

**Comment by a-kud** [comment]
I'm not sure why some unrelated snapshots got updated.

File: src/routes/landing/components/Heading.js (hunk 3 above)
Comment (maxammann) [reply src/routes/landing/components/Heading.js]: You can remove it as it is no longer used. Should be easy to add again afterwards.

File: src/modules/common/components/__tests__/__snapshots__/Page.spec.js.snap [hunk 4]
```diff
@@ -19,7 +19,7 @@ exports[`Page should render 1`] = `
   />
   <Translate(LastUpdateInfo)
     language="en"
-    lastUpdate={"2016-01-07T09:36:24.000Z"}
+    lastUpdate={"2016-01-07T08:36:24.000Z"}
```
Comment (maxammann) [inline src/modules/common/components/__tests__/__snapshots__/Page.spec.js.snap]: We need to do something about this :) I'll create an other task for this. So you do no need to change anything here.

The reason why these changes were necessary *for you* is the difference in the timezone. We need to switch to UTC here.

File: src/locales.json [hunk 5]
```diff
@@ -42,7 +42,7 @@
         "search": "جستجو",
         "events": "تازه​ها",
         "extras": "اضافات",
-        "landing": "تو کجایی؟",
+        "landing": "انتخاب محل",
```
Comment (maxammann) [inline src/locales.json]: If you are not a native farsi speaker remove this for now :)

File: src/locales.json [hunk 6]
```diff
@@ -54,7 +54,7 @@
         "search": "Rechercher",
         "events": "Actualités",
         "extras": "Extras",
-        "landing": "Tu es où?",
+        "landing": "Sélection de l'emplacement",
```
Comment (maxammann) [inline src/locales.json]: If you are not a native french speaker remove this for now :)

**Comment by maxammann** [comment]
Uuiui the reason why the build is failing is that our build server has a CET as timezone :dagger: 
Just revert the shapshot changes and the build server will build just fine. (The build will fail locally which is fine for now)

I'll do the task about the timezones now to fix this.

**Comment by maxammann** [comment]
Feel free to merge :)

**Pull request merged**